OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -651,7 +651,8 @@ void SelectMap(MapPreview map)
 
 						server.SyncLobbyInfo();
 
-						server.SendLocalizedMessage(ChangedMap, Translation.Arguments("player", client.Name, "map", server.Map.Title));
+						var player = LobbyUtils.SanitizePlayerName(client);
```

Comment:
Server messages censor your own name

Reply:
I did not find a way to get the local client because the interfaces are inaccessible here.

Reply:
Can we send the client ID instead of the name and do the replacement on the client side? Also, how does the server know which setting the client has enabled? 🤔 Sanitization should only happen on the client side.